bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -209,18 +213,33 @@ if args.timeslide_interval is None:
 
 data = {'stat':[], 'decimation_factor':[], 'timeslide_id':[], 'template_id':[]}
 
-for tnum in range(tmin, tmax):
-    tids={}
+if args.randomize_template_order:
+    seed(0)
+    template_ids = numpy.arange(0, num_templates)
+    shuffle(template_ids)
+    template_ids = template_ids[tmin:tmax]
+else:
+    template_ids = range(tmin, tmax)
+
+for tnum in template_ids:
+    tids = {}
     for ifo in trigs:
-        tids[ifo]=trigs[ifo].set_template(tnum)
+        tids[ifo] = trigs[ifo].set_template(tnum)
 
     for ifo in tids:
-        if len(tids[ifo] == 0):
+        if not len(tids[ifo]):  # no triggers in this template
             continue
 
-    times={}
+    times = {}
     for ifo in trigs:
-        times[ifo]=trigs[ifo]['end_time']
+        times[ifo] = trigs[ifo]['end_time']
+        snr = trigs[ifo]['snr']
+        # Coinc finding relies on triggers being spaced by more than coinc window
+        cid = coinc.cluster_over_time(snr, times[ifo], TWOEARTH)
+        if len(snr) - len(cid):
+            times[ifo] = times[ifo][cid]
```

Comment:
I doubt this does the right thing in terms of preserving the single trig info.

Reply:
now given up on doing this as it seems it would be excessively complicated to do the right book-keeping for a rare corner case